pkg/apis/operator/v1alpha1/constants.go

```diff
@@ -28,4 +28,7 @@ const (
 	// OperationRotateWorkloadIdentityKeyComplete is a constant for an annotation on a Shoot indicating that the
 	// rotation of the workload identity signing key shall be completed.
 	OperationRotateWorkloadIdentityKeyComplete = "rotate-workload-identity-key-complete"
+
+	// VirtualGardenNamePrefix is a constant to prefix various resource names for the virtual garden.
+	VirtualGardenNamePrefix = "virtual-garden-"
```

Comment:
I think it would be better to define here constants for the components names instead of computing them every time they need to be used

Something like:


```suggestion
	VirtualGardenNamePrefix = "virtual-garden-"
	// DeploymentNameVirtualGardenKubeAPIServer is a constant for the name of a Kubernetes deployment object that contains
	// the kube-apiserver pod for the virtual Garden cluster.
	DeploymentNameVirtualGardenKubeAPIServer = VirtualGardenNamePrefix + v1beta1constants.DeploymentNameKubeAPIServer
	...
```

Reply:
Sure, I added them in [374e2e7](https://github.com/gardener/gardener/pull/13635/commits/374e2e736c48e6d4da704f469fdc196f2330f0e1).